bin/test-workflow/7_app_deploy.sh

```diff
@@ -13,7 +13,7 @@ check_env_var SECRETS_PROVIDER_TAG
 check_env_var SECRETLESS_BROKER_TAG
 
 # Upon error, dump kubernetes resources in the application Namespace
-# trap dump_application_namespace_upon_error EXIT
+trap dump_application_namespace_upon_error EXIT
```

Comment:
If we want to keep this in, should probably change this to a trap on `ERR` so we don't dump logs every run.

Reply:
It looks like the dump_application_namespace_upon_error function checks whether there's an error. I'm not sure why it's done this way but that's the way it is in all the other scripts to it's probably worth keeping consistent.